MPF.md

```diff
@@ -632,12 +632,12 @@ defined. Bundles tagged with ```autorun``` are **not required** to be placed in
 
 #### Additional automatically loaded inputs
 
-When `def.mpf_additional_auto_input_dirs` is defined (and services_autorun is defined), the policy files (`*.cf`) in those directories will be added to inputs. If a directory is specified but is not a directory, it will be skipped.
+When `def.mpf_extra_autorun_inputs` is defined (and services_autorun is defined), the policy files (`*.cf`) in those directories will be added to inputs. If a directory is specified but is not a directory, it will be skipped.
```

Comment:
I think there should be a link here for how to add `.cf` policy files to inputs.

Reply:
ACK